test/README.md

```diff
@@ -86,17 +86,3 @@ You must specify a remote host for XDebug to connect to via the `XDEBUG_REMOTE_H
 
 - Running the entire integration test suite with XDebug enabled sometimes results in a segmentation fault. This appears to be an issue with XDebug itself.
 
-## Quick install for Developer in Ubuntu
```

Comment:
We should put a note for line 11 to use the latest version of docker-compose or at least a recent one
V 1.8.1 is old.

Reply:
@PapillonMcGill has this been resolved? i see `sudo apt install docker-compose` in the code. does this satisfy your comment? please give approving review if yes

Reply:
Nope,
Near the top, the first instruction is to `curl` an (quite) older version of docker. 
It is only in the instruction for developer that instruction is to `sudo apt install docker-compose`

`curl` is ok but should not point to a old version. Should have text to download the latest version from docker site.

Reply:
@PapillonMcGill could you suggest a change directly on here that @kongtiaowang could apply and commit?

Reply:
-Next, install Docker Compose: 
+Next, install Docker Compose, replacing <version> by the latest version of docker-compose available at https://github.com/docker/compose/releases.

curl -L https://github.com/docker/compose/releases/download/1.8.1/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+ curl -L https://github.com/docker/compose/releases/download/<version>/docker-compose-`uname -s`-`uname -m` > ~/docker-compose

Reply:
@PapillonMcGill If you go to the 'Files changed' tab, and click on the line(s) you want changed as if to add a comment, you can use the 'Insert a suggestion' feature to make a suggested change that shen can then directly commit on GitHub. This will make changing the code faster and easier!

<img width="633" alt="screenshot 2018-11-20 12 33 35" src="https://user-images.githubusercontent.com/17415878/48792074-55600700-ecc1-11e8-8104-fd4435b5219c.png">

Reply:
@zaliqarosli not in this case as those lines haven't been modified nor are adjacent to modified lines by this PR.

Reply:
@PapillonMcGill ahh i see, that's unfortunate :(